packages/grid-pro/test/edit-column-renderer.common.js

```diff
@@ -180,10 +180,11 @@ describe('edit column renderer', () => {
       };
 
       dblclick(cell._content);
-      expect(spy.calledOnce).to.be.true;
+      await nextFrame();
```

Comment:
Necessary because of the timing changes. Also, the test fails with the existing code when this is added. Focus is always triggered twice.